src/mapi/glapi/gen/es_EXT.xml

```diff
@@ -795,6 +795,14 @@
     <enum name="FRAGMENT_SHADER_DISCARDS_SAMPLES_EXT" value="0x8A52"/>
 </category>
 
+<!-- 150. GL_EXT_sRGB -->
```

Comment:
The extension requires OGLES 1 and above. 
Anyways, as it is in the else if block the !_mesa_is_gles3(ctx)  can be removed.